docs/source/documentation/team/wow-2.html.md.erb

```diff
@@ -0,0 +1,222 @@
+---
+owner_slack: "#data-platform-notifications"
+title: Ways of Working - Apps and Tools
+last_reviewed_on: 2023-10-31
+review_in: 2 months
+---
+
+# <%= current_page.data.title %>
+
+## Introduction
+
+We are the Data Platform Apps and Tools team (DPAT), a mix of developers and engineers. Our job is twofold: keeping our current [Analytical Platform](https://user-guidance.analytical-platform.service.justice.gov.uk/) running, and developing the future of Data and Analysis within the organisation, [Data Platform](https://technical-documentation.data-platform.service.justice.gov.uk/).
```

Comment:
I dont think `Data and Analysis` should be capitalised unless it is formal term we are using that I am not aware of?